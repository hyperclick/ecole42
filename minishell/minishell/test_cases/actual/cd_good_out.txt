/mnt/c/git/e49/minishell/minishell
/home/darugula
/mnt/c/git/e49/minishell/minishell
