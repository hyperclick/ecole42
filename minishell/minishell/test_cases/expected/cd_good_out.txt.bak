/mnt/c/git/e59/minishell/minishell
/home/darugula
/home/darugula
/home/darugula
