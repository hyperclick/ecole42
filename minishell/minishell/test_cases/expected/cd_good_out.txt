/mnt/c/git/e57/minishell/minishell
